vpr/src/route/rr_graph.cpp

```diff
@@ -284,16 +286,18 @@ static void alloc_and_load_tile_rr_graph(RRGraphBuilder& rr_graph_builder,
 static float pattern_fmod(float a, float b);
 static void load_uniform_connection_block_pattern(vtr::NdMatrix<int, 6>& tracks_connected_to_pin,
```

Comment:
Add Doxygen comment of purpose and parameters (as much info as you know at least).